tests/golden_tests.rs

```diff
@@ -471,6 +471,21 @@ fn io() {
   })
 }
 
+/// Runs a file that uses the prelude.
+#[test]
+fn prelude() {
+  run_golden_test_dir(function_name!(), &|code, path| {
+    let _guard = RUN_MUTEX.lock().unwrap();
+    let book = parse_book_single_file(code, path)?;
+    let compile_opts = CompileOpts::default();
+    let diagnostics_cfg = DiagnosticsConfig::default();
```

Comment:
```suggestion
    let diagnostics_cfg = DiagnosticsConfig::new(Severity::Error, true);
```
There shouldn't be any warnings in the prelude library, so we can just error on them